app/scripts/metamask-controller.js

```diff
@@ -1295,6 +1299,17 @@ export default class MetamaskController extends EventEmitter {
 
     ///: END:ONLY_INCLUDE_IF
 
+    ///: BEGIN:ONLY_INCLUDE_IF(platform-notifications)
```

Comment:
Have you considered a runtime check here rather than a code fence? e.g.

```
if (process.env.PLATFORM_NOTIFICATIONS) {
```

Runtime checks are much easier to work with than code fences (no fence-specific lint errors that only get highliged as part of the build, easier to test these conditions in unit tests, etc.)